src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php

```diff
@@ -139,6 +170,6 @@ private static function getTimezones()
             $timezones[$region][str_replace('_', ' ', $name)] = $timezone;
         }
 
-        return $timezones;
+        return 1 === count($timezones) ? reset($timezones) : $timezones;
```

Comment:
We get a flat array (1 region) or a nested one (multiple regions). Let me explain with a picture :)

![image](https://user-images.githubusercontent.com/1047696/28566811-90f869aa-7130-11e7-954f-64b216fe4217.png)

It's just cosmetic sugar. IMHO.